### PR TITLE
annotation: fine tune the comment permission in mobile

### PR DESCRIPTION
**Pull request by lpranam**
Signed-off-by: Pranam Lashkari <lpranam@collabora.com>
Change-Id: I6ec564d2524a9cc8735179fc282eaf8deb81a74b

* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/control/Control.JSDialogBuilder.js [hunk 1]
```diff
@@ -2365,14 +2366,6 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		annotation.setExpanded();
 		annotation.hideMarker();
 		annotation.sectionProperties.annotationMarker = null;
-
-		var replyCountNode = document.getElementById('reply-count-node-' + data.id);
-		if (replyCountNode)
-			replyCountNode.style.display = 'none';
-
-		var arrowSpan = document.getElementById('arrow span ' + data.id);
```
Comment (eszkadev) [inline browser/src/control/Control.JSDialogBuilder.js:2373]: this code is removed by mistake I think?

Comment (lpranam) [reply browser/src/control/Control.JSDialogBuilder.js:2373]: no that was intentional.

**Comment by eszkadev** [comment]
when opened PDF in desktop mode - "Insert comment" was not present in the insert menu

**Comment by eszkadev** [comment]
in mobile view in PDF I had no "insert comment" or "reply"

**Comment by eszkadev** [comment]
This PR has to be tested together with: https://github.com/CollaboraOnline/online/pull/5426

**Review by gokaysatir — APPROVED** [review]

**Pull request merged**